src/m_menu.c

```diff
@@ -4245,6 +4237,14 @@ setup_menu_t gen_settings6[] = {
 
   {"", S_SKIP, m_null, M_X, M_SPC},
 
+  {"View Bobbing", S_THERMO|S_PCT, m_null, M_X_THRM8, M_SPC,
+   {"view_bobbing_pct"}},
+
+  {"Weapon Bobbing", S_THERMO|S_PCT, m_null, M_X_THRM8, M_THRM_SPC,
+   {"weapon_bobbing_pct"}, 0, M_UpdateCenteredWeaponItem},
```

Comment:
Here are my suggestions: As mentioned above, move the thermos to "Weapons > Cosmetic" and make them the first two items. Then switch the order of the "Preferences" and "Cosmetic" tabs so the two thermos are the first things the user sees upon entering the submenu. These are the most important settings in the "Weapons" submenu since they relate to motion sickness, so let's make them easier to find. Finally, consider using "bob" instead of "bobbing" for all the variable names and strings. It looks cleaner and it's what a lot of old games used for naming.

Reply:
Also, the `M_X_THRM8` thermo can't handle 100 steps when using the mouse and 1% increments seems unnecessary for view/weapon bob. I suggest `0% 25% 50% 75% 100%` with `M_X_THRM4` thermos.

Reply:
I agree with your suggestions @ceski-1 !

Reply:
I tried a few more ideas and in practice I think a slight modification of Alaux's original layout is very good and makes the most sense to me as well:

<details><summary>Example:</summary>

![woof0000](https://github.com/fabiangreffrath/woof/assets/56656010/7a943034-bb86-45b5-9cfa-6eeb650dc6de)

![woof0001](https://github.com/fabiangreffrath/woof/assets/56656010/19d7012e-8fcf-4382-b760-c3da750bb00f)

![woof0002](https://github.com/fabiangreffrath/woof/assets/56656010/90fa7976-71fc-4365-a648-12a40d5004a6)

</details>

Reply:
> Also, the `M_X_THRM8` thermo can't handle 100 steps when using the mouse and 1% increments seems unnecessary for view/weapon bob. I suggest `0% 25% 50% 75% 100%` with `M_X_THRM4` thermos.

I still recommend this change to the thermos.

Reply:
I agree with the thermo-size and name changes. As for the layout, though, I'd like to wait for Fabian and Roman's input.

Reply:
I still believe that "Weapon Bob" and "View Bob" should be next to each other, preferably in  "Weapons" menu.

By the way, "Level Brightness" is not an accessibility feature; it originally came from the Unity port, where it's the "Video" option:

![image](https://github.com/fabiangreffrath/woof/assets/5077629/2064ec29-a007-40d2-9a0f-1d7ae2e79b81)

Maybe we should move it to "Video" as well?

Reply:
That all sounds reasonable to me.

Reply:
> I still believe that "Weapon Bob" and "View Bob" should be next to each other, preferably in "Weapons" menu.

Yes, this was a single feature and now got split into two, which are still closely connected. And even though "View Bob" might not directly relate to the "Weapons" section, it still affects the first person view, i.e. the "weapon's point of view".

Reply:
So:

- Replace "bobbing" with "bob";
- Make thermos smaller;
- Move thermos to `Weapons -> Cosmetic`.

Is that it?

I'm not sure if swapping the Weapons pages and moving the Level Brightness item are changes that have been fully decided upon, and even then, they seem out of scope for this PR, so I'd rather not make those changes myself.

Reply:
Level Brightness is a different issue. 

I don't care if swapping the Weapons menu tabs should be done in this PR or a later one, but I agree that it should be done.

Reply:
> * Make thermos smaller;

We also want bigger steps `0% 25% 50% 75% 100%`. I can do it in new PR (and brightness too).